ignite/pkg/cosmosgen/generate_composables.go

```diff
@@ -0,0 +1,167 @@
+package cosmosgen
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/imdario/mergo"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/ignite/cli/ignite/pkg/cosmosanalysis/module"
+	"github.com/ignite/cli/ignite/pkg/gomodulepath"
+)
+
+type composablesGenerator struct {
+	g            *generator
+	FrontendType string
```

Comment:
Does it need to be public?

```suggestion
	frontendType string
```

The `FrontendType` references must be updated to `frontendType` if this change is applied.

Reply:
good point...I guess it doesn't? trying to think if there is a scenario where it might need to be